### ISSUE TITLE
Persistent Subscriptions/ Competing consumer

### ISSUE THREAD

**Issue by brendanwheble** [x-cubed/event-store-client#37]
I'm looking to add this functionality. Are there any technical reasons why this hasn't been done? Or is it simply a missing feature?

**Comment by x-cubed** [comment]
No, there's no reason why it can't be added, and there are other people that want it too.
I simply haven't needed that feature, so I haven't added it.

Feel free to send in a pull request.

Thanks
